examples/gazebo_scene_viewer/GazeboDemo.cc

```diff
@@ -18,19 +18,19 @@
 #include <gazebo/common/Console.hh>
 #include <gazebo/transport/TransportIface.hh>
 
-#include <ignition/rendering.hh>
+#include <gz/rendering.hh>
 
 #include "CameraWindow.hh"
 #include "SceneManager.hh"
 
-using namespace ignition;
+using namespace gz;
 using namespace rendering;
 
 void Connect()
 {
-  gazebo::common::Console::SetQuiet(false);
-  gazebo::transport::init();
-  gazebo::transport::run();
```

Comment:
Oh this is actually Gazebo classic

Reply:
oh I think these lines should not be removed. This demo shows integration with gazebo classic so the old gazebo-classic calls are expected